parser/prism/Translator.cc

```diff
@@ -354,11 +354,35 @@ std::unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::Args>(parser.translateLocation(loc), std::move(params));
         }
+        case PM_PARENTHESES_NODE: { // A parethesized expression, e.g. `(a)`
+            auto parensNode = reinterpret_cast<pm_parentheses_node *>(node);
+
+            if (auto stmtsNode = parensNode->body; stmtsNode != nullptr) {
+                auto inlineIfSingle = false;
+                return translateStatements(reinterpret_cast<pm_statements_node *>(stmtsNode), inlineIfSingle);
+            } else {
+                return make_unique<parser::Begin>(parser.translateLocation(&parensNode->base.location), NodeVec{});
+            }
```

Comment:
This is really similar to the code in `PM_EMBEDDED_STATEMENTS_NODE`. I imagine that `PM_BEGIN_NODE` and other "`begin`-like" grouping nodes will have a similar pattern.

I'll extract a helper to duplicate this later, once I have more info of what the other cases will look like.